src/api/VotingIrregularities.Api/appsettings.json

```diff
@@ -13,7 +13,7 @@
     "UseHttps": true
   },
   "ConnectionStrings": {
-    "DefaultConnection": "TBR"
```

Comment:
Don't commit changes to `appsettings.json`. You can use `appsettings.development.json` for you local environment.

Reply:
Wasn't supposed to commit there, I must've missed it or something as I was already using appsettings.development.json, anyways, it should be 'clean' now.